app.js

```diff
@@ -1,13 +1,23 @@
 const express = require("express");
+const listEndpoints = require("express-list-endpoints");
+
 const { getAllTopics } = require("./controller/topics.controller");
-const { handleCustomErrors } = require("./errors");
+const { handleCustomError, handleServerErrors } = require("./errors");
 
 const app = express();
 
 app.use(express.json());
 
 app.get("/api/topics", getAllTopics);
 
-app.get("/api/nvnewbs", getAllTopics);
+app.get("/api", (req, res) => {
+  // console.log(listEndpoints(app));
+  res.status(200).send(listEndpoints(app));
+});
+
+app.use(handleCustomError);
+app.use(handleServerErrors);
+
+app.all("/*", getAllTopics);
```

Comment:
This is invoking `getAllTopics` when the user puts in an incorrect endpoint. 
Instead, you should have a separate function for handling this, which sends off a 404 and an error message